sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -544,10 +585,24 @@ private synchronized void setInstanceListener(AnalyticsListener listener) {
     }
 
     @Override
-    public synchronized void onStarted(@NonNull Context context, String appSecret, String transmissionTargetToken, @NonNull Channel channel) {
-        super.onStarted(context, appSecret, transmissionTargetToken, channel);
+    public synchronized void onStarted(@NonNull Context context, @NonNull Channel channel, String appSecret, String transmissionTargetToken, boolean startFromApp) {
```

Comment:
`startedFromApp` would be more consistent with the other variables. Is it intentional or a typo?

Reply:
I can rename